ubuntuStation/templates/user_form.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% block content %}
+<div class="well">
+{% from "_formhelpers.html" import render_field %}
+{{ form.csrf_token }}
+<script src="{{ url_for('static', filename='jquery.sortable.js') }}"></script>
+<h1>Add User</h1>
```

Comment:
Adding new users page should be mentioned in the original issue. I was expecting only the login page to have been changed. Others may not be aware for the same reason.

The name of the file also does not portray its purpose. `create_user.html` as the form action name suggests would be better and avoid conflicts in the long run with other file names should we decide to create other forms associated to users.